### ISSUE TITLE
Mac 2.3.2 Binaries reference odd location for tbb library

### ISSUE THREAD

**Issue by jakewendt** [BenLangmead/bowtie2#128]
I just tried to upgrade from 2.2.7 to 2.3.2. I saw that I had to install TBB, so I did via MacPort.
```
$ ls -l /opt/local/lib/libtbb*
-rwxr-xr-x 1 root 272124 Jul 28 12:34 /opt/local/lib/libtbb.dylib
-rwxr-xr-x 1 root 102384 Jul 28 12:34 /opt/local/lib/libtbbmalloc.dylib
-rwxr-xr-x 1 root  14388 Jul 28 12:34 /opt/local/lib/libtbbmalloc_proxy.dylib
```
I downloaded the precompiled binaries and attempted to run...
```
$ ./bowtie2
dyld: Library not loaded: /usr/local/opt/tbb/lib/libtbb.dylib
  Referenced from: /Users/jakewendt/Downloads/bowtie2-2.3.2/bowtie2-align-s
  Reason: image not found
(ERR): Description of arguments failed!
Exiting now ...
```
I don't understand why it is looking in /usr/local/opt/tbb/lib/. /usr/local/opt does not exist and I can find no reference to it in any of my environment variables.

I have since downloaded and compiled the source which runs fine so this is more of an FYI.

**Comment by chinanicole** [comment]
Ran into this issue as well and just installed TBB via homebrew (brew install tbb) then could run bowtie2 fine

**Comment by ch4rr0** [comment]
We recently released v2.3.3 which has TBB statically linked to the bowtie2* binaries. This should resolve the dylib issue.

https://github.com/BenLangmead/bowtie2/releases/download/v2.3.3/bowtie2-2.3.3-macos-x86_64.zip

**Comment by BenLangmead** [comment]
Resolved in recent releases; let us know if it's still an issue
